header/main.scss

```diff
@@ -2,127 +2,126 @@
 @import 'scss/marketing-promo';
 @import '../typeahead/main';
 
-@if nUiApply('header') {
 
-	@include nUiCritical('core') {
+@include nUiStylesheetStart('head-n-ui-core');
 
-		@include oHeader(('subnav', 'search', 'nav', 'anon', 'megamenu', 'simple'));
-		// force display of search row
-		// <https://github.com/Financial-Times/o-header/blob/2ebf2cbe6b7508a03e0989ff57f1029e12c0506c/src/scss/rows/_search.scss#L13-L15>
-		.core [data-o-header-search] {
-			display: block;
-		}
-
-		// inelegant gymnastics to fix focus outline of the first breadcrumb link
-		.o-header__subnav-wrap-outside {
-			margin-left: -2px;
-		}
-		.o-header__subnav-list--breadcrumb .o-header__subnav-item:first-child .o-header__subnav-link {
-			margin-left: 2px;
-		}
-
-		// add next opt-out buttons
-		.o-header__drawer-menu-button {
-			@include oButtons('medium', 'standout');
-			display: block;
-			margin: $_o-header-drawer-padding-x;
-		}
-
-		.o-header__mega {
-			@include nUtilZIndexFor('meganav');
-		}
-
-		.o-header__drawer,
-		.o-header--sticky {
-			display: none;
-		}
-
-		// skip links for keyboard users
-		.n-skip-link {
-			@include nUtilZIndexFor('notification');
-			position: absolute;
-			top: 1em;
-			left: -9999px;
-			padding: 0.5em 1em;
-			background: getColor('purple-1');
-			color: getColor('white');
-
-			&:focus {
-				@include oEffectsShadowsElevation('high');
-				left: 0;
-				color: getColor('white');
-			}
-		}
-
-		.o-header__search-close {
-			margin-bottom: -3px;
-		}
-
-		@include nUiHeaderMarketingPromo;
-		.n-typeahead {
-			display: none;
-		}
+	@include oHeader(('subnav', 'search', 'nav', 'anon', 'megamenu', 'simple'));
+	// force display of search row
+	// <https://github.com/Financial-Times/o-header/blob/2ebf2cbe6b7508a03e0989ff57f1029e12c0506c/src/scss/rows/_search.scss#L13-L15>
+	.core [data-o-header-search] {
+		display: block;
 	}
 
-	@include oHeaderDrawer;
-	@include oHeaderSticky;
-	@include nTypeahead;
-	// fix up z-indexes
-	.o-header__drawer {
-		@include nUtilZIndexFor('drawer');
+	// inelegant gymnastics to fix focus outline of the first breadcrumb link
+	.o-header__subnav-wrap-outside {
+		margin-left: -2px;
 	}
-	.o-header--sticky {
-		@include nUtilZIndexFor('sticky-header');
+	.o-header__subnav-list--breadcrumb .o-header__subnav-item:first-child .o-header__subnav-link {
+		margin-left: 2px;
 	}
 
-	.o-header__drawer,
-	.o-header--sticky {
+	// add next opt-out buttons
+	.o-header__drawer-menu-button {
+		@include oButtons('medium', 'standout');
 		display: block;
+		margin: $_o-header-drawer-padding-x;
 	}
 
-	.o-header__drawer__viewtoggle {
-		display: flex;
-		align-items: center;
-		padding: 0 16px;
-		font-size: 18px;
-		width: 100%;
-		box-sizing: border-box;
-		border-top: 1px solid #cec6b9;
-		border-bottom: 1px solid #cec6b9;
-
-		.o-header__viewtoggle__label {
-			padding: 12px 12px 12px 0;
-		}
+	.o-header__mega {
+		@include nUiZIndexFor('meganav');
```

Comment:
In discussions with matt h we thought it'd be a good idea to allow for a small family of `n-ui` things (certainly having the lowest level utilities in a separate component is useful), but we don't particularly want long namespaces, so letting anything in an `n-ui-*` component use the `nUi` namespace

So the short answer is, it's in the `n-ui-foundations` component